src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final int NAME_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
+    public static final String DELIMITER = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> readDataList = readDataFromFile(fromFileName);
+        String dataToWrite = valuesCounter(readDataList);
+        writeDataInFile(toFileName, dataToWrite);
+    }
+
+    private List<String> readDataFromFile(String dataFile) {
+        File file = new File(dataFile);
+        List<String> readList;
+        try {
+            readList = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file " + dataFile, e);
+        }
+        return readList;
+    }
+
+    private void writeDataInFile(String resultFile, String data) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(resultFile))) {
+            bufferedWriter.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write in file " + resultFile, e);
+        }
+    }
+
+    private String valuesCounter(List<String> linesFromFile) {
```

Comment:
```suggestion
    private String countValues(List<String> linesFromFile) {
```
method names should be verbs

Reply:
replaced